zerver/models.py

```diff
@@ -3193,6 +3193,14 @@ def where_starred() -> str:
         # will generate a query involving `flags & 2 = 2`, which doesn't match our index.
         return "flags & 2 <> 0"
 
+    @staticmethod
+    def where_flag_is_present(flag_int: int) -> str:
+        return f"flags & {1 << flag_int} <> 0"
```

Comment:
Given that we're the maintainers and most of the active contributors to this bitfield library, and have been since 2017:
https://github.com/disqus/django-bitfield/commits/master

another thing we could perhaps do here is fix the issue explained in that comment on `where_starred`, so that the nice clean syntax `.filter(flags=flagattr)` (or perhaps some more-explicit variation) would Just Work. But that's out of scope for this PR.